packages/create-audius-app/index.ts

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env node
+import { cyan, green, red, bold } from 'picocolors'
+import Commander from 'commander'
+import path from 'path'
+import prompts from 'prompts'
+import { createApp, DownloadError } from './create-app'
+import { validateNpmName } from './helpers/validate-pkg'
+import packageJson from './package.json'
+import { isFolderEmpty } from './helpers/is-folder-empty'
+import fs from 'fs'
+
+let projectPath: string = ''
+
+const handleSigTerm = () => process.exit(0)
+
+process.on('SIGINT', handleSigTerm)
+process.on('SIGTERM', handleSigTerm)
+
+const program = new Commander.Command(packageJson.name)
+  .version(packageJson.version)
+  .arguments('<project-directory>')
+  .usage(`${green('<project-directory>')} [options]`)
+  .action((name) => {
+    projectPath = name
+  })
+  .option(
+    '-e, --example [name]',
+    `
+
+  An example to bootstrap the app with. You can use an example name
+  from the Audius repo. They are found in AudiusProject/audius-protocol/packages/libs/src/sdk/examples
+  For example: --example react
+`,
+    'react'
+  )
+  .allowUnknownOption()
+  .parse(process.argv)
+
+async function run(): Promise<void> {
```

Comment:
nit but do we need explicit returns?

Reply:
You mean the return type? No I don't think we need it